### PR TITLE
NEX-65: Add image fields to Page and Frontpage, and improve descriptions for Excerpt fields.

### PR DESCRIPTION
**Pull request by Tromppa**
https://wunder.atlassian.net/browse/NEX-65

*Changes proposed in this PR:*
- Improve the description of Excerpt field so that editors understand it's used for Search and Metatags also. 
- Add image fields to Frontpage and Page so that they will be used in the og:image and search. 

*Link to feature environment:*
https://feature-nex-65-next.next4drupal-project.dev.wdr.io/

### CODE REVIEW

File: drupal/recipes/wunder_pages/config/field.field.node.page.field_image.yml [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+    translation_sync:
+      alt: alt
+      title: title
+      file: '0'
+id: node.page.field_image
+field_name: field_image
+entity_type: node
+bundle: page
+label: 'Search/Meta Image'
+description: 'Image used in the search results and for Open Graph html meta tags, so that sharing the content looks good.'
```
Comment (vermario) [inline drupal/recipes/wunder_pages/config/field.field.node.page.field_image.yml:22]: The search results do not display images at the moment :)

Comment (Tromppa) [reply drupal/recipes/wunder_pages/config/field.field.node.page.field_image.yml]: Is it coming soon, or am I just predicting into the wild :D but isn't it though? In eg. google searches :)

Comment (vermario) [reply drupal/recipes/wunder_pages/config/field.field.node.page.field_image.yml]: it could be a good next feature for the search actually :)

**Review by vermario — APPROVED** [review]
Works! 

I have pushed a change for the excerpt description, if you like it I think it's good to go.

**Comment by vermario** [comment]
Since that image is not required, I'd say let's not add it to the migration this time.

**Pull request merged**